api/src/pcapi/alembic/versions/20240119T143931_23e2d3b322ab_create_debit_note_reference_scheme.py

```diff
@@ -0,0 +1,21 @@
+"""
+create debit note reference scheme
+"""
+
+from alembic import op
+
+
+revision = "23e2d3b322ab"
+down_revision = "d3bd3af52558"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("INSERT INTO public.reference_scheme VALUES (6, 'debit_note.reference', 'A', 2024, 1, 7);")
+    op.execute("INSERT INTO public.reference_scheme VALUES (7, 'debit_note.reference', 'A', 2025, 1, 7);")
+    op.execute("INSERT INTO public.reference_scheme VALUES (8, 'debit_note.reference', 'A', 2026, 1, 7);")
```

Comment:
[détail] L'id est auto-incrémenté, il n'est donc pas nécessaire. De plus, je trouve qu'il est plus lisible d'indiquer le nom des colonnes (notamment pour les valeurs 1 et 7 qui ne sont pas explicites).

```suggestion
    for year in (2024, 2025, 2026):
        op.execute(
            f"""
            INSERT INTO public.reference_scheme
            (name, prefix, year, "nextNumber", "numberPadding")
            VALUES ('debit_note.reference', 'A', {year}, 1, 7);
        """
        )
```

Je note que lorsque la migration est écrite ainsi, on n'a plus d'erreur dans `tests/core/reference/test_models.py`.